bigquery/tests/unit/test_table.py

```diff
@@ -675,6 +694,7 @@ def test_create_w_alternate_client(self):
         RESOURCE['expirationTime'] = _millis(self.EXP_TIME)
         RESOURCE['view'] = {}
         RESOURCE['view']['query'] = QUERY
+#        RESOURCE['view']['useLegacySql'] = True
```

Comment:
Why is this line commented out?  (and likewise, the ones below for the same testcase).  If we really want this testcase *not* to pass the value, then they should just be removed.